packages/valory/skills/apy_estimation_abci/ml/forecasting.py

```diff
@@ -341,6 +342,9 @@ def predict_safely(forecaster: Pipeline, steps_forward: int) -> Any:
     """
     try:
         y_hat = forecaster.predict(steps_forward)
+    except NotFittedError:
```

Comment:
Where is it handled?

Reply:
I chose to re-raise and not handle it since this would have been a developer's error, i.e., using a non-fitted model to predict.